### PR TITLE
doc: clarify dataDirHostPath must be unique for each cluster

### PR DESCRIPTION
**Pull request by satoru-takeuchi**
If there are multiple clusters, `dataDirHostPath` must be unique for each cluster. However, it's not documented yet. In addition, `ceph-teardown.md` was also updated because the original document assumes that `dataDirHostPath` is `/var/lib/rook` and the files for a cluster `rook-ceph` is under `/var/lib/rook/rook-ceph`.

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by travisn — APPROVED** [review]
Want to also add a comment about this in cluster.yaml with the example setting?

File: Documentation/Storage-Configuration/ceph-teardown.md [hunk 1]
```diff
@@ -88,11 +88,11 @@ kubectl delete -f crds.yaml
 ### Delete the data on hosts
 
 !!! attention
-    The final cleanup step requires deleting files on each host in the cluster. All files under the `dataDirHostPath` property specified in the cluster CRD will need to be deleted. Otherwise, inconsistent state will remain when a new cluster is started.
+    The final cleanup step requires deleting files on each host in the cluster. All files under `dataDirHostPath` property specified in the cluster CRD will need to be deleted. Otherwise, inconsistent state will remain when a new cluster is started.
```
Comment (travisn) [inline Documentation/Storage-Configuration/ceph-teardown.md:91]: Let's add back `the` in this sentence, it is a small difference, but slightly better language.

**Review by travisn — COMMENTED** [review]
Just one nit, then looks good to merge, thanks! Let's also confirm the CI will be green after a fresh push. I don't see how the CI issues are related, but if they are still failing after pushing again, let's investigate more.

**Comment by travisn** [comment]
Commonly doc changes are only backported to the latest release, but could be backported to 1.14 if you think necessary.

File: Documentation/Storage-Configuration/ceph-teardown.md (hunk 1 above)
Comment (satoru-takeuchi) [reply Documentation/Storage-Configuration/ceph-teardown.md]: Fixed, thanks. English is really difficult for me :-)

**Review by travisn — APPROVED** [review]

**Pull request merged**